### ISSUE TITLE
Settings - Notifications - Search doesn't function

### ISSUE THREAD

**Issue by s3m1s0n1c** [DevLAN-io/RFSPager-issues#7]
* Search in Brigade selection box doesn’t work

**Comment by nicko170** [comment]
Yeah disabled for now, as full text search engine has no way to sort by date, and I cannot give the unsorted elements to the views yet :(
